### ISSUE TITLE
Improve display of untrustworthy sentences

### ISSUE THREAD

**Issue by alanfgh** [Tatoeba/tatoeba2#326]
CK thinks that the CSS for untrustworthy sentences (sentences with "correctness" == -1) should be marked with strikethrough and a slightly dark background, not just with a red font color. He sent an e-mail with this idea to the Tatoeba team.

In his words, "line-through and a slightly dark background could make it much clearer that the red is there as a negative warning and not to highlight something important."

**Comment by jiru** [comment]
> Initially when I was coding the feature I wanted to go for grey instead of red, but the problem was that indirect translations are also greyish, and I didn't want people to be confused about sentences being displayed in grey even though they are not indirect translations.

What changed your opinion? [This](https://dev.tatoeba.org/sentences/show/224003) is rather confusing in my opinion.

**Comment by trang** [comment]
I didn't really change my opinion. Back then my problem was that I didn't want to implement something more than just changing the text color, so I chose to set the text color in red despite the fact that it was too catchy in my opinion. In my last changes I added a warning icon.

I'll be asking for feedbacks on the Wall anyway, but what confuses you exactly? Is it because the warning icon is not noticeable enough and it feels like the unapproved sentence is just another indirect translation?
